COCOAPODS.md

```diff
@@ -212,8 +605,8 @@ directive.
 2. Add the path to your Xcode project `Podfile` with `podfile = project.file(..)` to `build.gradle.kts` (`build.gradle`) 
```

Comment:
Could you add this as an open question to discuss on our sync – how we format `build.gradle.kts` (`build.gradle`)?
In JS docs, we format as `build.gradle`(`.kts`). Let's agree on consistent formatting.